packages/eslint-config/eslint-builtin-rules.json

```diff
@@ -1,50 +1,19 @@
 {
-    "array-bracket-newline": "off",
```

Comment:
Note that the rules removed here were [already being disabled](https://github.com/prettier/eslint-config-prettier/blob/5687e7de69ac132f053b4bc5474d5963aa3bf6d6/index.js) by `eslint-config-prettier`, and will now be enforced by Prettier.